src/notification-hub/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+1.0.0b2
+++++++
+* Remove `az notification-hub test-send`
```

Comment:
You may need to upgrade the major version as it's a breaking change

Reply:
done